Adding /hackathon buttons

eparubrub opened the pull request:
- adding ticket button, discord button, and dropdown button with social links
- adding a new dropdown component 
- updating tech spec image

eparubrub committed:
adding buy tickets button
- adding new dropdown with social links
- updating tech spec image

eparubrub committed:
adding custom width for smaller screens

eparubrub committed:
forgot to convert px to rem

ulu5 commented on pages/hackathon.tsx:
```diff
@@ -87,33 +92,49 @@ export default function HackathonPage({ pageTitle }) {
-        </DataList> */}
+          <Dropdown
+            links={{
+              "purplemaia.org": "https://purplemaia.org/",
```
Is it possible to add the Twitter and Instagram icons here to disambiguate the two? https://fontawesomeicons.com/instagram

ulu5 commented on pages/hackathon.tsx:
```diff
@@ -87,33 +92,49 @@ export default function HackathonPage({ pageTitle }) {
+              "@purple_maia": "https://twitter.com/purple_maia/",
+              purple_maia: "https://www.instagram.com/purple_maia/",
+              "@hawaiiansintech": "https://twitter.com/hawaiiansintech/",
+              hawaiiansintech: "https://www.instagram.com/hawaiiansintech/",
```
Is it possible to add the Twitter and Instagram icons here to disambiguate the two? https://fontawesomeicons.com/instagram

ulu5 commented on components/Dropdown.tsx:
```diff
@@ -0,0 +1,83 @@
+          font-weight: 400;
+          padding: 1rem 2.2rem;
+          // TODO: Figure out why the css below is not showing
+          /* border-bottom: 1rem solid ${theme.color.background.alt}; */
```
Hmm...I wonder what the issue is here. A horizontal separator would be ideal especially if it could be added to separate purple maia links from hawaiians in tech ones.

ulu5 commented on components/Dropdown.tsx:
```diff
@@ -0,0 +1,83 @@
+          /* border-bottom: 1rem solid ${theme.color.background.alt}; */
+        }
+
+        /* .list-item:nth-last-child(1) {
```
Did you mean to leave this in?

ataeoalii commented on components/Dropdown.tsx:
```diff
@@ -0,0 +1,83 @@
+          /* border-bottom: 1rem solid ${theme.color.background.alt}; */
+        }
+
+        /* .list-item:nth-last-child(1) {
```
Did you mean to leave this in?

eparubrub replied:
removed in latest commit

ataeoalii commented on components/Dropdown.tsx:
```diff
@@ -0,0 +1,83 @@
+          cursor: pointer;
+          font-weight: 400;
+          padding: 1rem 2.2rem;
+          // TODO: Figure out why the css below is not showing
```
Hmm...I wonder what the issue is here. A horizontal separator would be ideal especially if it could be added to separate purple maia links from hawaiians in tech ones.

eparubrub replied:
fixed in latest commit

ataeoalii commented on pages/hackathon.tsx:
```diff
@@ -87,33 +92,49 @@ export default function HackathonPage({ pageTitle }) {
+              "@purple_maia": "https://twitter.com/purple_maia/",
+              purple_maia: "https://www.instagram.com/purple_maia/",
+              "@hawaiiansintech": "https://twitter.com/hawaiiansintech/",
+              hawaiiansintech: "https://www.instagram.com/hawaiiansintech/",
```
Is it possible to add something to disambiguate between the twitter/IG links? Not sure if we can add font-aweseome icons https://fontawesomeicons.com/instagram.

eparubrub replied:
added in latest commit

ataeoalii reviewed (commented):
Put in some suggestions, let me know what you think. All pretty minor, ok to approve if you think it's better or easier to keep it as is.

ataeoalii reviewed (approved):
Marking approved to unblock merge.

eparubrub committed:
adding twitter and IG icons for links
- fixing horizontal separator in dropdown
- removing comments from dropdown component
